### ISSUE TITLE
Cross Site Scripting(Many places)

### ISSUE THREAD

**Issue by OldFRE** [sitewhere/sitewhere#660]
Software version：sitewhere-server-1.13.0
Download address：http://www.sitewhere.org/downloads/ 
OS ： win

1. first visit and login.
http://127.0.0.1:8080/sitewhere/admin/

2. copy URL and access, resulting in reflective cross site scripting vulnerabilities.

http://127.0.0.1:8080/sitewhere/admin/default/specifications/rf5qx%3Cimg%20src=a%20onerror=alert(1)%3Exmxoh

http://127.0.0.1:8080/sitewhere/admin/default/groups/jypc9%3Cimg%20src=a%20onerror=alert(1)%3Ej5ryx

http://127.0.0.1:8080/sitewhere/admin/default/sites/lvwq2%3cimg%20src%3da%20onerror%3dalert(1)%3esf5ae

http://192.168.169.132:8080/sitewhere/admin/default/devices/fn5fn<img src%3da onerror%3dalert(1)>gtx15

3. printscreen

![a6eb2260-ca7b-4a4b-9d51-6aa8af34e3dc](https://user-images.githubusercontent.com/19726132/46611472-84bb1a00-cb40-11e8-90e2-af523ceece4d.png)
![40244153fbb8f251a26b11e432a3cebf](https://user-images.githubusercontent.com/19726132/46611516-9d2b3480-cb40-11e8-9e2e-7f2c9c484eb2.jpg)
![38893559-8a9f-44dc-9639-2c6ea473047e](https://user-images.githubusercontent.com/19726132/46611525-a3211580-cb40-11e8-8301-92d9fd88f051.png)
![c0eddc36-5daa-4eed-a7f7-6ad6b120a6fe](https://user-images.githubusercontent.com/19726132/46611650-eda29200-cb40-11e8-84b6-42ee75d1d906.png)

**Comment by derekadams** [comment]
Yes, the old jQuery user interface is not something that should ever be public facing since it was only designed for system administrators and is not locked down. In SiteWhere 2.0 we have changed to Vue as the rendering technology and wrapped the admin application as an Electron app that does not allow the URLs to be manipulated. If you are using 1.13, the admin application should be secured so that only a system administrator can access it.

**Comment by OldFRE** [comment]
Thank you for your reply. I came from your official website and submitted questions. He has just discovered that he is 2. Ha-ha. I'm sorry. It seems that you have made some thoughts and protection on its safety. I just discovered this problem by accident, and I will pay you back in time. Because he's really good software. So I will submit it to you to see if it is helpful to you. Since it only visits administrators, it does not do much harm.
I hope I am not disturbing your work. I am really sorry! I wish you a happy life .  ; )
